boards/xtensa/m5stack_core2/m5stack_core2.dts

```diff
@@ -125,6 +125,7 @@
 	axp192_pmic: axp192@34 {
 		compatible = "x-powers,axp192";
 		reg = <0x34>;
+		/delete-node/ vbusen-disable; /* make sure manual control VBUS is enabled */
```

Comment:
why is this needed? where is this boolean property set?

Reply:
I wanted to explicitly make sure, this option is false. That isn't required?

Reply:
no, it's not

Reply:
Fixed in https://github.com/zephyrproject-rtos/zephyr/commit/17544d699aa18a4226001f50cab33aad23c8d225